src/App.js

```diff
@@ -14,11 +14,67 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selector: false,
+    close: false,
+  };
+
+  closeButton = (text) => {
+    this.setState({ selector: false });
+    this.setState({ close: false });
+    document.querySelector('.App__list--active').className = 'App__list';
+  }
+
+  writeSrelector = (event, value) => {
+    this.setState({ selector: value });
+    this.setState({ close: true });
+    const key = event.target;
+
+    const active = document.querySelector('.App__list--active');
+
+    if (active !== null) {
+      active.className = 'App__list';
+    }
+
+    key.className = 'App__list App__list--active';
+  }
+
+  render() {
+    const { selector, close } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          Selected good: -
+          {' '}
+          { selector }
+          <span>
+            <button
+              type="button"
+              className={close && 'App__close'}
+              onClick={this.closeButton}
+            >
+              {close}
+            </button>
```

Comment:
```suggestion
            {!!selector && <button
              type="button"
              className={close && 'App__close'}
              onClick={this.closeButton}
            >
              {close}
            </button>}
```
if something is selected, then button is showing up, if it's false in selector, button will not show up